service.subtitles.napisy24pl/resources/lib/NapisyUtils.py

```diff
@@ -145,7 +145,7 @@ def download(self, id):
         for file in files:
             if os.path.splitext(file)[1] in exts:
                 filename_dest = os.path.join(__temp__, file)
-                flag = xbmcvfs.copy(os.path.join(zip_filepath, file), filename_dest)
+                flag = xbmcvfs.copy(zip_filepath + '/' + file, filename_dest)
```

Comment:
why does this fix anything?

Reply:
It was test on Windows machine by a user (I don't have a PC).
Since this line tries to extract a file from zip file (via `zip://` protocol), probably it requires forward slash

Reply:
Hum.. maybe you want to have a look at: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcvfs.html#ga7d325903d7bc797f993c9df791d22c44

Reply:
How this method helps me? 
When coping from zip protocol, the slash needs to be `/`, no reason to "validate" it

Reply:
The name is probably not the best choice ever, but it seems to return a validated path not exactly a bool. I'm fine accepting this as is, just seems a hack to me...that's all.

Reply:
Hehe, bad name!
I will ask the user to check if it works for him, if so, i will update

Reply:
for some reason zip:// path requires a forward slash after archive name (and before the archive contents).
os.path.join() adds backslash on windows and that causes the extraction to fail

Reply:
@enen92 can you merge it as is?